sdk/sdk-api/src/main/java/mil/emp3/api/KML.java

```diff
@@ -232,4 +232,17 @@ public IGeoDocument getGeoDocument() {
     public String exportToKML() {
         return "";
     }
+
+    /**
+     * Convert all features to String
+     * @return
+     */
+    @Override
+    public String toString() {
+        String str = "Feature " + getFeatureType();
+        for(IFeature f: getFeatureList()) {
+            str += f.toString();
+        }
+        return str;
+    }
```

Comment:
This can get very long. We have seen KML with thousands of features.  Why not return the name and the number of features contained. Something more predictable.